package.json

```diff
@@ -23,11 +23,13 @@
     "@material-ui/styles": "^4.11.4",
     "@testing-library/dom": "^8.11.3",
     "@testing-library/jest-dom": "^5.16.2",
+    "lodash": "^4.17.21",
```

Comment:
Because our data starts as JSON data anyway, could we do a JSON roundtrip to handle deep clones? Maybe we can break this out into our own `deepClone` function? I don't think that we have any functions or non-JSONable data types.

```js
export function deepClone(obj) {
  return JSON.parse(JSON.stringify(obj));
}
```